docs/reference/tab-widgets/troubleshooting/data/increase-node-capacity.asciidoc

```diff
@@ -0,0 +1,299 @@
+the {es} cluster.
+
+//end::kibana-api-ex-2[]
+
+//tag::increase-azs[]
+[increase-azs]
+Now that you know the tier, increase the number of availability zones.
+
+. Go back to the deployment's landing page by clicking on the three horizontal bars on the top left of the screen
+and choosing **Manage this deployment**. On that page click the **Manage** button, and choose **Edit deployment**.
+
+. In the {es} section, find the tier where the replica shards could not be assigned. Select a higher number
+of availability zones for that tier.
++
+
+[role="screenshot"]
+image::images/data-tiers/ess-advanced-config-data-tiers.png[{kib} Console,align="center"]
+//end::increase-azs[]
+
+If it is not possible to increase the number of availability zones, you can reduce the number of replicas of your index
+data. We'll achieve this by inspecting the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting
+index setting and decreasing the configured value.
+
+//tag::kibana-api-ex-2[]
+. Access {kib} as described above.
+
+. Inspect the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting.
++
+[source,console]
+----
+GET /my-index-000001/_settings/index.number_of_replicas
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "my-index-000001" : {
+    "settings" : {
+      "index" : {
+        "number_of_replicas" : "2" <1>
+      }
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
++
+<1> Represents the currently configured value for the number of replica shards
+required for the index
+
+. Use the `_cat/nodes` API to find the number of nodes in the target tier:
++
+[source,console]
+----
+GET /_cat/nodes?h=node.role
+----
+// TEST[continued]
++
+The response will look like this, containing one row per node:
++
+[source,console-result]
+----
+himrst
+mv
+himrst
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
++
+You can count the rows containing the letter representing the target tier to know how many nodes you have. See
+<<cat-nodes-api-query-params>> for details. The example above has two rows containing `h`, so there are two
+nodes in the hot tier.
++
+. <<indices-update-settings, Decrease>> the
+value for the total number of replica shards required for this index. The new value needs to be less
+than the number of nodes found above minus one. Since the example above found 2 nodes in the hot tier,
+the maximum value for `index.number_of_replicas` is 1.
++
+[source,console]
+----
+PUT /my-index-000001/_settings
+{
+  "index" : {
+    "number_of_replicas" : 1 <1>
+  }
+}
+----
+// TEST[continued]
+
++
+<1> The new value for the `index.number_of_replicas` index configuration
+is decreased from the previous value of `2` to `1`. It can be set as low as 0 but
+configuring it to 0 may lead to temporary availability loss during node restarts
+or permanent data loss in case of data corruption.
```

Comment:
```suggestion
is decreased from the previous value of `2` to `1`. It can be set as low as 0 
but configuring it to 0 for indices other than <<searchable-snapshots,
searchable snapshot indices>> may lead to temporary availability loss during 
node restarts or permanent data loss in case of data corruption.```